tutorials/katacoda/basic-pipeline/step1.md

```diff
@@ -13,9 +13,11 @@ spec:
   steps:
     - name: hello
       image: ubuntu
-      script: |
-        set -e
-        echo "Hello World!"
```

Comment:
The reason the commands are being output is that the script doesn't specify a shebang. See https://tekton.dev/docs/pipelines/tasks/#running-scripts-within-steps for details, essentially this is running with `set -ex`.

Updating this to add `#!/usr/bin/env bash` should prevent the commands being output while keeping the YAML reasonably short. Also not sure why `set -e` is included in such a simple example, it could be removed leaving just:

```
script: |
  #!/usr/bin/env bash
  echo "Hello World!"
```

Reply:
Thanks, that explains the behaviour (actually pretty much alike GitLab CI’s handling). I’ll change it back to using the `script` key, to keep it small.